internal/tiltfile/tiltextension/plugin.go

```diff
@@ -126,9 +129,30 @@ func (e *Plugin) LocalPath(t *starlark.Thread, arg string) (localPath string, er
 //
 // If it has, returns the existing object (which should only have a spec).
 //
+// If it hasn't, but it's a nested extension name, and the root has been registered, register it and
+// return it.
+//
 // Otherwise, infers an extension object that points to the default repo.
 func (e *Plugin) ensureExtension(t *starlark.Thread, objSet apiset.ObjectSet, moduleName string) *v1alpha1.Extension {
+	// Copy the supplied module name (which may contain slashes) into the "repo path".
+	// This ensures that sanitizing or name munging does not effect the path where the extension
+	// will be loaded.
+	repoPath := moduleName
+
+	// Before sanitizing the name, split off anything after the first /
+	extRoot, nestedPath, isNested := strings.Cut(moduleName, "/")
```

Comment:
i don't think it's correct to assume that if a module name has a slash, then it's nested in another extension, here's an extension this would break:
https://github.com/tilt-dev/tilt-extensions/tree/master/tests/golang#usage

Reply:
How would that break? It'd load the nested `golang` extension underneath the `tests` extension.

Also, I hadn't even realized that there were already nested extensions in the default repo. I'll take a look at that and adjust.